### DISCUSSION TITLE
Perms for Town.

### DISCUSSION THREAD

**Discussion by MishutkaOgnev** [TownyAdvanced/Towny#7091] [Q&A] [answered]
Hey there! I need your help because I'm struggling with something. I've been dealing with this for a year now, searched the entire internet, read all the documentation for the Towny plugin, but I can't find anything. Let's get to the point:

I have 2 towns, Westervale and Valtro. How can I set different permissions for the residents of these towns? For example, I want residents of Valtro to be able to use the /spawn command, but residents of Westervale shouldn't. How can I make that happen?

**Comment by LlmDl** [comment, marked as answer]
Each town is given a node like towny.town.{townname} (Documented [here](https://github.com/TownyAdvanced/Towny/wiki/Towny-Permission-Nodes#miscellaneous-nodes))

So if you set up in your permission plugin that node: towny.town.westervale which inherits some child nodes, the member of those towns ought to get their child nodes.
